packages/cli/src/commands/db.ts

```diff
@@ -68,16 +68,17 @@ const runMigrateUp = async ({silent = false} = {}) => {
   return runPrismaGeneration({silent})
 }
 
-export const runMigrate = async (name?: string) => {
+export const runMigrate = async (flags: object = {}) => {
   if (process.env.NODE_ENV === "production") {
     return runMigrateUp()
   }
+  // @ts-ignore escape:TS7053
+  const nestedFlags = Object.keys(flags).map(key => [`--${key}`, flags[key]])
+  const options = ([] as string[]).concat(...nestedFlags)
```

Comment:
shouldn't this do the same?

```suggestion
  const options = nestedFlags as string[]
```

Reply:
Thank you reviewed!

Fixed it! https://github.com/blitz-js/blitz/pull/1012/commits/1e062d209612ed49fd64a3e4eb2838bc01a5f6c4 ⚡️

Reply:
I forgot to mention that it's a fix for making it a flat because it's a nest array.

Before:
```ts
  const nestedFlags = Object.keys(flags).map(key => [`--${key}`, flags[key]])
  const options = ([] as string[]).concat(...nestedFlags)
  
  console.debug('options', options)

=>     options [ '--name', 'name' ]
````

https://github.com/blitz-js/blitz/commit/1e062d209612ed49fd64a3e4eb2838bc01a5f6c4:

````ts
const formatFlags = Object.keys(flags).map(key => [`--${key}`, flags[key]])
  const options = formatFlags as unknown as string[]
  
  console.debug('options', options)

=>    options [  [ '--name', 'name' ] ] 
````

Reply:
I reverted.
Are there any corrections to this?